### PR TITLE
Migrate controlfreec pytest to nf-test

### PR DESCRIPTION
**Pull request by maxulysse**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/sarek _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: tests/variant_calling_controlfreec.nf.test [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+    //                 // All stable path name, with a relative path
+    //                 stable_name,
+    //                 // All files with stable contents
+    //                 stable_path,
+    //                 // All cram files
+    //                 cram_files.collect{ file -> [ file.getName(), cram(file.toString(), fasta).getReadsMD5() ] }
+    //             ).match() }
+    //         )
+    //     }
+    // }
```
Comment (FriederikeHanssen) [inline tests/variant_calling_controlfreec.nf.test:141]: why is all of this commented?

Comment (maxulysse) [reply tests/variant_calling_controlfreec.nf.test:141]: I can remove them if you prefer, I let them for now, just for a reminder to sort this out later at some point

Comment (FriederikeHanssen) [reply tests/variant_calling_controlfreec.nf.test:141]: why are we not doing the stub test then?

Comment (maxulysse) [reply tests/variant_calling_controlfreec.nf.test:141]: `Cannot invoke method clone() on null object`

Comment (FriederikeHanssen) [reply tests/variant_calling_controlfreec.nf.test]: hm that seems like we are calling something strange somewhere?

**Review by FriederikeHanssen — APPROVED** [review]

**Review by matthdsm — APPROVED** [review]

**Pull request merged**